components/index/SlideBackground.js

```diff
@@ -29,7 +29,7 @@ export default class SlideBackground extends React.Component {
     render() {
         return (
             <StyledSlideBackground
-                image={this.props.image}
+                image={this.props.image['large']}
```

Comment:
The problem here is that the image is set as background image instead of a `img` tag, so that `background-size: cover` can be used to to stretch the image as needed. Any idea if that can be solved in a different way, so that an `img` tag with `srcset` can be used? /cc @leph83 @tomsvogel